rev_news/drafts/edition-52.md

```diff
@@ -270,4 +270,5 @@ Christian Couder &lt;<christian.couder@gmail.com>&gt;,
 Jakub Narębski &lt;<jnareb@gmail.com>&gt;,
 Markus Jansen &lt;<mja@jansen-preisler.de>&gt; and
 Gabriel Alcaras &lt;<gabriel.alcaras@telecom-paristech.fr>&gt;
-with help from David Pursehouse and Johannes Schindelin.
+with help from Jeff Hostetler, David Pursehouse and
```

Comment:
with help from jeff Hostetler,zakir

Reply:
Jakub Narębski &lt;<jnareb@gmail.com>&gt;,
Markus Jansen &lt;<mja@jansen-preisler.de>&gt; and
Gabriel Alcaras &lt;<gabriel.alcaras@telecom-paristech.fr>&gt;
with help from David Pursehouse and Johannes Schindelin.
with help from Jeff Hostetler, David Pursehouse and